tools/check_ip_addresses.pl

```diff
@@ -44,9 +44,9 @@ END
   next if $ip =~ /198\.51\.100\./;      # TEST-NET-2 (RFC 5737)
   next if $ip =~ /203\.0\.113\./;       # TEST-NET-3 (RFC 5737)
   next if $ip =~ /198\.(18|19)\./;      # BMWG (RFC 2544)
-  next if $ip =~ /20\.0\./;             # 20.0.0.1/15
-  next if $ip =~ /30\.0\./;             # 30.0.0.1/15
-  next if $ip =~ /100\.0\./;            # 100.0.0.1/12
+  next if $ip =~ /20\./;              # 20.0.0.1/15
+  next if $ip =~ /30\./;              # 30.0.0.1/15
```

Comment:
ain't you change the allowed subnet from `/15` to `/16`?

Reply:
I have used /15 in 14.2, but its giving error. It looks like pattern matching is looking for 20.0  ? Hence thought of changing it. Please let me know any other suggestions.

IPBlockEncapA           = "20.1.64.1/15"  
	IPBlockEncapB           = "20.5.64.1/15" 
	IPBlockEncapC           = "20.10.64.1/15"
	IPBlockEncapD           = "20.15.64.1/15"

Reply:
@cprabha  range of `20.0/15` only contains `20.0.0.1 - 20.1.255.254`, right? 

https://github.com/openconfig/featureprofiles/blob/da4eaf8d4609fb279b84a1eae6ed63fc07c9795b/CONTRIBUTING.md?plain=1#L363

Reply:
> @cprabha range of `20.0/15` only contains `20.0.0.1 - 20.1.255.254`, right?
> 
> https://github.com/openconfig/featureprofiles/blob/da4eaf8d4609fb279b84a1eae6ed63fc07c9795b/CONTRIBUTING.md?plain=1#L363

Yeah, for scale scripts it not sufficient. WE need to change this to /8 to accommodate the scale addresses.

Reply:
I have updated comment also contributing.md with /8, please check.

Reply:
updated 20/30/100 to Martian address range.